packages/calcite-components/src/components/handle/handle.tsx

```diff
@@ -180,14 +180,21 @@ export class Handle implements LoadableComponent, T9nComponent, InteractiveCompo
   // --------------------------------------------------------------------------
 
   /**
-   * Emitted when the handle is activated and the up or down arrow key is pressed.
+   * Emits whenever the component is selected or unselected.
```

Comment:
@geospatialem @DitwanP do we have a convention on "Emits" vs "Emitted"? I've seen them both throughout the code base. Maybe we should pick one and do some cleanup in a different PR? I'm partial to "Emits" because we use "Specifies" for properties.

Reply:
Good question, we've moved towards "Emits" for the reason you specify above, to keep present tense on the docs. It looks like there are two outliers in `list` and `list-item-group`. Otherwise there is mention of "emitted" throughout the code, but not in the doc. Added this note to our conventions for #7071.